### PR TITLE
[GR-1040] Add "Approve this run in MISO" button to single lane pages

### PR DESCRIPTION
**Pull request by asofmosq**
In order to stop the page from re-loading when a new value was added to runs list, I wrapped the button in a core.Loading but it slows down the page some on initial load. If there's a way to accomplish this without the slowdown, please let me know

### CODE REVIEW

**Comment by asofmosq** [comment]
It's also fastest on the single lane WGS page but considerably slower on the other two

**Comment by avarsava** [comment]
image
Button layout gets squashed when my screen is at 50% width (my usual use case for Dashi), could we use a relative width somewhere to fix this?

**Review by avarsava — COMMENTED** [review]
This seems more reactive than the rest of the page - if it decided whether to display the button only when Update is pressed, could that fix the issue with needing the separate Loading() for it? Users would probably want to see the graphs for the run before approving it anyway

File: application/dash_application/views/single_lane_rna.py [hunk 1]
```diff
@@ -533,6 +535,17 @@ def update_pressed(click,
...
+    @dash_app.callback(
+        [Output(ids["approve-run-button"], "href"),
+         Output(ids["approve-run-button"], "style")],
+        [Input(ids["run-id-list"], "value")]
+    )
+    def approve_run(value):
+        if len(value) == 1:
+            return ["https://miso.oicr.on.ca/miso/run/alias/" + value[0], {"display": "inline-block"}]
+        else:
+            return ["", {"display": "none"}]
```
Comment (callunity) [inline application/dash_application/views/single_lane_rna.py]: If you wanted, you could move this functionality into sidebar_utils and use one method everywhere.

**Review by callunity — APPROVED** [review]

**Review by avarsava — APPROVED** [review]
Introduces a tiny display bug when window width is at 50%:
image
But that can be addressed later

**Review by callunity — APPROVED** [review]

**Pull request merged**